mcda/utils/utils_for_main.py

```diff
@@ -308,11 +320,40 @@ def check_input_matrix(input_matrix: pd.DataFrame) -> pd.DataFrame:
     return input_matrix_no_alternatives
 
 
+def ensure_directory_exists(path):
+    """
+    Ensure that the directory specified by the given path exists.
+    If the directory does not exist, create it and any intermediate directories as needed.
+
+    Parameters:
+        path (str): The path of the file in the directory to ensure exists.
+
```

Comment:
Would you also like to add information about throwing exception here in the documentation?

```
Raises:
    - Exception: 
```